requirements.txt

```diff
@@ -1,8 +1,12 @@
+connexion==2.7.0
```

Comment:
Please check whether you really need all of these packages, e.g., I'm not sure what you would need `flask-cors` for, or even `connexion` and `Flask`. Only add what you _explicitly_ import from. If packages are installed as dependencies of another module that you need explicitly, don't mention them here, because the version lock may cause unnecessary problems if you update dependencies.